desktop/apps/auction2/client.js

```diff
@@ -71,96 +65,39 @@ if (sd.AUCTION && sd.AUCTION.is_live_open == false) {
 }
 
 // Commercial filtering
-const params = new Params(defaultParams, {
-  customDefaults: defaultParams
-})
-const filter = new Filter({ params: params })
+const loggerMiddleware = createLogger()
+
+const store = createStore(
+  auctions,
+  applyMiddleware(
+    thunkMiddleware, // lets us dispatch() functions
+    loggerMiddleware // middleware that logs actions
+  )
+)
 
-// REDUX
-const store = createStore(auctions)
 render(
   <Provider store={store}>
     <HeaderContainer />
   </Provider>,
   document.getElementById('cf-view-filter')
-);
+)
+
+render(
+  <Provider store={store}>
+    <SidebarContainer />
+  </Provider>,
+  document.getElementById('cf-sidebar')
+)
```

Comment:
Once everything is built out we can probably turn this into a full single page app, where there's only one render being called, but we can worry about that way later. (I remember our previous discussion about how best to "componentize" these little bits for interop with existing BB code..)